script/deploy-debs.pl

```diff
@@ -52,17 +50,19 @@
 	print STDERR "WARNING: $passfile does not exist!  New RPMs will not be signed!";
 }
 
-opendir(FILES, '.') or die "Unable to read current directory: $!";
-while (my $line = readdir(FILES)) {
+my $FILEDIR_HANDLE;
+
+opendir($FILEDIR_HANDLE, '.') or die "Unable to read current directory: $!";
+while (my $line = readdir($FILEDIR_HANDLE)) {
 	next if ($line =~ /^\.\.?$/);
 	chomp($line);
 	if ($line =~ /\.deb$/) {
-		push(@FILES_DEBS, $line);
+		push(@$FILEDIR_HANDLE_DEBS, $line);
```

Comment:
I think this was mistakenly changed.

Reply:
I went ahead and pushed the fix for this in 09221ba03061f827f11745fc76aa389094aefa11.